build.gradle

```diff
@@ -173,11 +174,13 @@ dependencies {
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
   implementation group: 'com.github.hmcts', name: 'pip-data-models', version: '1.0'
   implementation 'org.postgresql:postgresql'
+  implementation 'org.springframework.boot:spring-boot-starter-webflux'
   implementation 'org.springframework.boot:spring-boot-starter-validation'
   implementation 'org.springframework.boot:spring-boot-starter-jdbc'
   implementation 'org.springframework.boot:spring-boot-starter-data-jpa:2.5.5'
   implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-core', version: '9.0.58'
   implementation group: 'org.apache.tomcat.embed', name: 'tomcat-embed-websocket', version: '9.0.58'
+  implementation("com.squareup.okhttp3:okhttp:4.9.3")
```

Comment:
try not to throw a rouge Kotlin gralde format in, keep the dependencies full gradle format

Reply:
didn't even realise this was a kotlin thing tbh! Will know for next time :)

Reply:
#the more you know ;)